src/ProjectSystemTools/LogModel/Builder/EvaluationInfo.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using Microsoft.Build.Framework;
+using Microsoft.VisualStudio.ProjectSystem.Tools.LogModel;
```

Comment:
Why was this added?

Reply:
It looks like it was because I didn't change namespaces of files while I was rearranging the projects, so it couldn't find the Resources.resx file without this using statement.

Reply:
Renamed the namespace, so I was able to remove these using statements